<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PyeongchangDialog</class>
 <widget class="QDialog" name="PyeongchangDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>767</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>410</y>
     <width>281</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QPushButton" name="btn_motion4">
      <property name="text">
       <string>Motion 4</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="btn_motion3">
      <property name="text">
       <string>Motion 3</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="btn_motion1">
      <property name="text">
       <string>Hand Shake</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="btn_motion2">
      <property name="text">
       <string>Bouquet</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="btn_motion6">
      <property name="text">
       <string>Motion 6</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="btn_motion7">
      <property name="text">
       <string>Motion 7</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="btn_motion8">
      <property name="text">
       <string>Motion 8</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="btn_motion9">
      <property name="text">
       <string>Motion 9</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="btn_motion10">
      <property name="text">
       <string>Motion 10</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="btn_motion5">
      <property name="text">
       <string>Motion 5</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="btn_enable">
      <property name="text">
       <string>All BTN Enable</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="btn_disable">
      <property name="text">
       <string>All BTN Disable</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>410</y>
     <width>160</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QPushButton" name="btn_Test1">
      <property name="text">
       <string>Low Gain Arm</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="btn_Test2">
      <property name="text">
       <string>High Gain Arm</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>531</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>HandShake</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>512</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btn_hand_ready">
       <property name="text">
        <string>Ready</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_hand_lg">
       <property name="text">
        <string>Low Gain</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_hand_gon">
       <property name="text">
        <string>grip on</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_hand_goff">
       <property name="text">
        <string>grip off</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_hand_hg">
       <property name="text">
        <string>High Gain</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_hand_init">
       <property name="text">
        <string>Init</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>261</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Bouquet</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="btn_bouquet_ready">
       <property name="text">
        <string>Ready</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_bouquet_give">
       <property name="text">
        <string>Give</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_bouquet_init">
       <property name="text">
        <string>Init</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>341</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Clap</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="btn_clap_ready">
       <property name="text">
        <string>Ready</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_clap_one">
       <property name="text">
        <string>One</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_clap_three">
       <property name="text">
        <string>Three</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_clap_korea">
       <property name="text">
        <string>korea</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_clap_init">
       <property name="text">
        <string>Init</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>250</y>
     <width>361</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Head</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>340</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QPushButton" name="btn_head_yes">
       <property name="text">
        <string>Yes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_head_no">
       <property name="text">
        <string>No</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_head_stretching">
       <property name="text">
        <string>Stretching</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_head_init">
       <property name="text">
        <string>Init</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>160</y>
     <width>431</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Bow</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>411</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QPushButton" name="btn_bow_one">
       <property name="text">
        <string>One Hand</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_bow_two">
       <property name="text">
        <string>Two Hand</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_bow_two_w">
       <property name="text">
        <string>Two w/w</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_bow_greeting">
       <property name="text">
        <string>Greeting</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_walkready">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>10</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Walk Ready</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_homepos">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>60</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Home Pos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_openal">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Open AL</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_CLAP_CNT">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>340</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_leg_mode">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
